stdlib/json/0/json.rbs

```diff
@@ -969,7 +969,7 @@ module JSON
   #        #<Admin:0x00000000064c41f8
   #        @attributes={"type"=>"Admin", "password"=>"0wn3d"}>}
   #
-  def self?.load: (string | _JsonReadableIO | _JsonRead source, ?Proc proc, ?json_options options) -> untyped
+  def self?.load: ((string | _JsonReadableIO | _JsonRead) & Kernel source, ?Proc proc, ?json_options options) -> untyped
```

Comment:
Is the `& Kernel` for `respond_to?`?

https://github.com/ruby/ruby/blob/master/ext/json/lib/json/common.rb#L537-L552